Karl/claire.h

```diff
@@ -439,7 +456,10 @@ class GenericClaire final : private DummyClaireNotifiable {
         }
       }
     } else if (r.method == "POST") {
-      if (r.url.query.has("report_to") && !r.url.query["report_to"].empty()) {
+      if (r.url.query.has("send_keepalive")) {
+        ForceSendKeepalive();
+        r("OK\n");
```

Comment:
Nit: If this is about to be used from outside the unit test, a chunked response with two chunks, `"Sending a forced keepalive to %s.\n"` and `"Done.\n"` could prove itself useful.